geowebcache-webapp/src/main/java/org/georchestra/geowebcache/security/PreAuthFilter.java

```diff
@@ -47,45 +49,60 @@
     public static final String SEC_USERNAME = "sec-username";
     public static final String SEC_ROLES = "sec-roles";
 
+    public PreAuthFilter() {
```

Comment:
Not related to this PR, but I am pretty sure that something in spring-security already exists which does basically the same as this class

Reply:
no freakin' idea TBH